### ISSUE TITLE
[3.3.5a] Ring of Valor - Elevators

### ISSUE THREAD

**Issue by Demonid** [TrinityCore/TrinityCore#14760]
Ring of Valor Elevators spawn at the Top instead of the bottom.

Here is a video to explain a little bit better:
https://www.youtube.com/watch?v=SuZIvGgXM0s

**Comment by DJScias** [comment]
Hello there, it'd be best if you about [the format of an issue](http://www.trinitycore.org/f/topic/37-the-trinitycore-issuetracker-and-you/).

You're missing information like the version of TDB you're using.

**Comment by Demonid** [comment]
This is the commit that breaks the elevators, maybe you could give it a check @Shauren

8b07f1cd4d1714379f8beeea3d13ea0af118de23

**Comment by DDuarte** [comment]
Are you sure it's that commit? The transport period changes are technically the same as before that commit....

**Comment by Demonid** [comment]
@DDuarte Yes, i just reverted that commit on a fresh TC and it works correctly.

**Comment by Shauren** [comment]
No, blame @DDuarte for random cherry picks, that commit is fully valid but only in 6.x, 3.3.5 transports behave a bit differently (this part https://github.com/TrinityCore/TrinityCore/commit/8b07f1cd4d1714379f8beeea3d13ea0af118de23#diff-fe5c8ebb7ad8e663836b00f39eb524d6R2224)

**Comment by Voidx** [comment]
still broken

**Comment by Aokromes** [comment]
This bug is still valid?

**Comment by tkrokli** [comment]
Perhaps we could get a confirmation from @Demonid , @Voidx or @Keader - since this is PvP related.

**Comment by Demonid** [comment]
I dont have a Clean Source to test right now, but taking into account that the commit on https://github.com/TrinityCore/TrinityCore/issues/14760#issuecomment-105429887 is still added and it should not be, the bug should still be there as @Voidx confirms.

**Comment by tkrokli** [comment]
Would be nice to have an updated confirmation _with TC rev. hash_ from someone who knows the Ring of Valor Elevators well, how it looks in-game now and how it is supposed to work if it was correct.

**Comment by Keader** [comment]
Another issue related with Ring of Valor.
Spells in area like Shadow Fury (Warlock) that need "click" on area for active, dont work in this elevators.
Spells return to server coord: 0,0,0 (x,y,z)

**Comment by ariel-** [comment]
> that commit is fully valid but only in 6.x, 3.3.5 transports behave a bit differently (this part 8b07f1c#diff-fe5c8ebb7ad8e663836b00f39eb524d6R2224)

Should it be reverted completely? Or partially?

**Comment by Demonid** [comment]
Still not working as of cf8380a3a77d2585f13945a53b7579a1d2fb8468
